.github/workflows/release.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Function Release
```

Comment:
We can use a better name here. Maybe "Create short tag for function release"?

Reply:
My thinking is that it makes sense to use the generic name for the pipeline, since the trigger which determines when the pipeline is run is associated with a function release. In the future I could see additional stages being added to this pipeline which do not have to do with tagging.

However, I think this wording could be adopted for the stage which creates the short tag - so I've updated the name of the stage accordingly.

Reply:
IMO this is not a complete release pipeline, since it doesn't actually build and publish the images.
If we want to use name `Function Release`, we should add a comment saying this is a partial release pipeline.

Reply:
That is a fair point, so this SGTM. I'll update the name